pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,109 @@
-schema = 1
-
-[device]
-name = "lightning.kokkos"
-
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
+schema = 2
 
 # The union of all gate types listed in this section must match what
 # the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "Identity",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "S",
-        "T",
-        "PhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CNOT",
-        "CY",
-        "CZ",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "IsingXX",
-        "IsingXY",
-        "IsingYY",
-        "IsingZZ",
-        "ControlledPhaseShift",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "SingleExcitation",
-        "SingleExcitationPlus",
-        "SingleExcitationMinus",
-        "DoubleExcitation",
-        "DoubleExcitationPlus",
-        "DoubleExcitationMinus",
-        "MultiRZ",
-        "QubitUnitary",
-        "GlobalPhase",
-]
+[operators.gates.native]
+
+CNOT = { properties = [ "invertible", "differentiable" ] }
```

Comment:
Is there a formatter for TOML that we can use to make this a little easier to read? Maybe some newlines and indenting? If not, we can keep as is for now.

Reply:
Indeed it might be overly verbose, but do you think that newlines will help in getting the whole picture? What do you think about table-like formatting, like
``` toml
DoubleExcitationPlus = { properties = [ "invertible",                       "differentiable" ] }
DoubleExcitation        = { properties = [ "invertible",                       "differentiable" ] }
GlobalPhase              = { properties = [ "invertible", "controllable", "differentiable" ] }
```
?

Reply:
Yep, honestly that looks good to me too. Having a quick 'n' easy way to scan it will make checks here easy. Thanks @grwlf

Reply:
@mlxd Done!